src/storage.rs

```diff
@@ -87,13 +110,14 @@ mod test {
     #[test]
     fn should_use_default_path() {
         let st = Storages::new(None);
-        assert_eq!(st.base_dir.as_os_str(), Path::new(DEFAULT_PATH).as_os_str());
+        assert_eq!(st.base_dir, default_path());
     }
 
     #[test]
     fn should_use_user_path() {
-        let user_path = Path::new("../some/path");
-        let st = Storages::new(Some(user_path));
-        assert_eq!(st.base_dir.as_os_str(), user_path.as_os_str());
+        let user_path: &str = "/tmp/some";
+        let st = Storages::new(Some(PathBuf::from(user_path)));
+
+        assert_eq!(st.base_dir, PathBuf::from(user_path));
```

Comment:
Unitests should not use real files, path or IO, it should be integration tests instead, i recommend to delete it completely.